acceptance/.shared/kitchen_acceptance/.kitchen.vagrant.yml

```diff
@@ -38,8 +38,10 @@ fedora-21
 # freebsd-8
 # ubuntu-12.04
 # centos-7
-# centos-6
-
+  - name: centos-6
+    driver:
+      box: opscode-centos-6.6
+      box_url: http://opscode-vm-bento.s3.amazonaws.com/vagrant/virtualbox/opscode_centos-6.6_chef-provisionerless.box
```

Comment:
Take a look at: https://github.com/chef-cookbooks/omnibus/blob/master/.kitchen.yml#L11-L35

You should not have to specify a box name or URL.